src/components/__tests__/ThoughtDelete.js

```diff
@@ -0,0 +1,20 @@
+import { act } from 'react-dom/test-utils'
+
+it('delete thought', async () => {
+  const thought = document.wrapper.find('div.editable')
+  expect(thought.text()).toBe('')
+  const keyboardResponder = document.wrapper.find('div#keyboard')
+  await thought.simulate('change', { target: { value: 'c' } })
+  await keyboardResponder.simulate('keydown', { key: 'Enter' })
+  jest.runAllTimers()
+  await thought.update()
+  await keyboardResponder.simulate('keydown', { key: 'Backspace' })
+  await thought.simulate('change', { target: { value: '' } })
+  await act(async () => {
+    await thought.simulate('keydown', { key: 'Backspace' })
+  })
+  jest.runAllTimers()
+  await document.wrapper.update()
+  const emptythoughts = document.wrapper.find('div.transformContain div ul.children')
+  expect(emptythoughts.length).toBe(0)
+})
```

Comment:
I am finding that these longer tests without any comments or spacing are a bit hard to read. Can you kindly add line breaks and comments in between major actions, such as "new thought", "edit thought", and "delete thought"?